### DISCUSSION TITLE
BUG?

### DISCUSSION THREAD

**Discussion by oliverhavrila** [ketilmo/balena-ads-b#89] [Q&A] [answered]
Hello,

I am reading, thinking and trying to set up it via the manual, but..

..check it out 

At first, I am not sure if this information are right..

![1](https://github.com/ketilmo/balena-ads-b/assets/28049460/30b7782d-7085-4172-8518-185c14deaa61)

also this one..

![2](https://github.com/ketilmo/balena-ads-b/assets/28049460/11dc3677-ea79-481b-80c0-3b9aa1ec21bd)

For example, I have set a new Flightradar24 receiver

after setup `cat /etc/fr24feed.ini` everything looks ok

but after last step: `24. Restart the fr24feed service under Services by clicking the "cycle" icon next to the service name.`

after restart  the fr24feed service, the config file is missing

```
root@xyz123:/# cat /etc/fr24feed.ini
cat: /etc/fr24feed.ini: No such file or directory
```

The result? My data sharing on flightradar24 = **Offline**

Can you help me, please?

Thanks

**Comment by oliverhavrila** [comment]
Through the CLI I tried this:

```
>balena fleets
ID      NAME         SLUG                   DEVICE TYPE     ONLINE DEVICES DEVICE COUNT
2096625 balena-ads-b name/balena-ads-b raspberrypi4-64 1              1

>balena push balena-ads-b
Error: no "Dockerfile[.*]", "docker-compose.yml" or "package.json" file found in source folder "."
```

**Comment by ketilmo** [comment]
Hi @oliverhavrila!

Thanks for reaching out and for using the **balena-ads-b** service. To better understand the issue, can you tell me how you deployed the setup to your device? Using the one-click approach at the [balena hub](https://hub.balena.io/) or using the `balena push` method via CLI? Also, can you confirm that you are running Raspberry Pi 4?

**Comment by santitf** [comment]
I just installed on a raspi 3B using Balena deploy and have the same with current release 0.0.0

**Reply by oliverhavrila** [reply]
Hello @ketilmo 
Thank you for your answer. I just create new, fleet, create new device all steps via [documentation](https://docs.balena.io/learn/getting-started/raspberrypi4-64/).

Confirm that I am using Raspberry Pi 4.

Now I see this:

<img width="793" alt="Screenshot 2023-11-24" src="https://github.com/ketilmo/balena-ads-b/assets/28049460/1eb023e1-019c-4e90-a650-1dce18229712">

And when I want to create release via CLI, I can't :/

```
C:\Windows\System32>balena fleets
ID      NAME     SLUG               DEVICE TYPE     ONLINE DEVICES DEVICE COUNT
xyz0123 airports o_havrila/airports raspberrypi4-64 1              1
```

```
C:\Windows\System32>balena push airports
Error: no "Dockerfile[.*]", "docker-compose.yml" or "package.json" file
found in source folder "."
```

So, some issue with docker?

Thank you

**Reply by oliverhavrila** [reply]
OK, after two glasses of wine I got this :)
```

[Success]          Release successfully created!
[Info]             Release: xyznumber (id: xyz)
[Info]             ┌─────────────────┬────────────┬────────────┬───────────────────────┐
[Info]             │ Service         │ Image Size │ Delta Size │ Build Time            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ kiosk           │ 732.11 MB  │ 0 bytes    │ < 1 second            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ wifi-connect    │ 12.96 MB   │ 0 bytes    │ < 1 second            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ frontend-proxy  │ 271.84 MB  │ 220.61 MB  │ 27 seconds            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ dump1090-fa     │ 182.66 MB  │ 139.00 MB  │ 1 minute, 11 seconds  │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ dump978-fa      │ 427.39 MB  │ 222.20 MB  │ 1 minute, 29 seconds  │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ piaware         │ 201.49 MB  │ 132.52 MB  │ 2 minutes, 59 seconds │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ fr24feed        │ 135.14 MB  │ 98.53 MB   │ 31 seconds            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ planefinder     │ 138.41 MB  │ 0 bytes    │ < 1 second            │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ opensky-network │ 170.01 MB  │ 0 bytes    │ 1 second              │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ radarbox        │ 242.57 MB  │ 0 bytes    │ 1 second              │
[Info]             ├─────────────────┼────────────┼────────────┼───────────────────────┤
[Info]             │ adsb-exchange   │ 243.84 MB  │ 2.67 MB    │ 1 minute,             │
[Info]             └─────────────────┴────────────┴────────────┴───────────────────────┘
[Info]             Build finished in 6 minutes, 18 seconds
                            \
                             \
                              \\
                               \\
                                >\/7
                            _.-(6'  \
                           (=___._/` \
                                )  \ |
                               /   / |
                              /    > /
                             j    < _\
                         _.-' :      ``.
                         \ r=._\        `.
                        <`\\_  \         .`-.
                         \ r-7  `-. ._  ' .  `\
                          \`,      `-.`7  7)   )
                           \/         \|  \'  / `-._
                                      ||    .'
                                       \\  (
                                        >\  >
                                    ,.-' >.'
                                   <.'_.''
                                     <'
```

After that I see this..

<img width="326" alt="Screenshot 2023-11-24 new" src="https://github.com/ketilmo/balena-ads-b/assets/28049460/e3b2594f-09f6-4108-9d29-ca2a939903e8">

I think that still not ok, because after restart fr24feed the config file is missing again :/

**Reply by ketilmo** [reply]
@santitf @oliverhavrila,

Thanks for sharing this. I recently made some changes to the fr24feeder, updating the OS in the container to a newer version. That works well on my end, but it might be the reason for the issue the two of you are seeing – which appears to be identical.

I have now reverted the OS updates. Please try re-pushing the code to update your installations and report back.

**Reply by ketilmo** [reply]
Hi @santitf,

Where you able to get up and running?

**Comment by oliverhavrila** [comment, marked as answer]
Hi @ketilmo,

thank you for your support! Everything started working after I set up variables LAT, LON and ALT in Device Variables. Probably I forgot for that at first.. strange 🤣😈

PS: now I am thinking, it is working with internet, only via wifi or lan? or will it be working also with usb modem or some sim card module for rpi?

PS: Merry Christmas 🧑‍🎄

Thank you

**Reply by ketilmo** [reply]
Hi @oliverhavrila,

Thanks for reporting back. I'm glad that you got it working. Getting it to work with a USB modem or a SIM card module might require some adjustments to the setup, but I haven't tried this myself. If you attempt this, please let us know how it goes.
